search-app/src/cmr/search/services/query_service.clj

```diff
@@ -301,6 +304,102 @@
      (ph/provider-holdings->string
        (:result-format params) provider-holdings {:echo-compatible? (= "true" echo-compatible)})]))
 
+(defn- get-collections-with-deleted-revisions
+  "Executes elasticsearch searches to find collections that have deleted revisions.
+   Returns a list of concept-ids of the collections."
+  [context params]
+  ;;1/ Find all collection revisions that are deleted satisfy the original search params
```

Comment:
It was from the `get-deleted-collections` function where it has steps 1/, 2/, 3/. I intentionally left it here to match that, but I think I will take it off now that it confuses you.